Automattic/wp-calypso
# Added more plugins to collections

#### Proposed Changes

* Added more plugins to collections

#### Testing Instructions



* Open [/the plugins page](http://calypso.localhost:3000/plugins) and make sure the new plugins are good suggestions and have the right copy on them

<img width="1181" alt="image" src="https://user-images.githubusercontent.com/1044309/195914152-2845312d-3986-4f63-9bce-102bdabe9a67.png">


<img width="1180" alt="image" src="https://user-images.githubusercontent.com/1044309/195914433-97baf1e9-3882-4842-977e-fccf9b6dcd8c.png">


Related to #68942
